### PR TITLE
Publish to PyPI upon GitHub release

### PR DESCRIPTION
**Pull request by charlotte-avery**
Following: https://www.seanh.cc/2022/05/21/publishing-python-packages-from-github-actions/

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: Publish to PyPI.org
+on:
+  release:
+    types: [published]
+jobs:
+  pypi:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```
Comment (shengy90) [inline .github/workflows/publish.yml]: Use the latest version @v4.

Comment (charlotte-avery) [reply .github/workflows/publish.yml]: Done

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        with:
+          fetch-depth: 0
+      - run: python3 -m pip install --upgrade build && python3 -m build
```
Comment (shengy90) [inline .github/workflows/publish.yml]: Use the same syntax for all steps:

```yml
- name: Build package
  run: |
     python3 -m pip install --upgrade build && python3-m build
```

**Review by shengy90 — APPROVED** [review]
Just small cosmetic changes but lgtm

**Pull request merged**